src/main/java/fi/helsinki/cs/tmc/core/TmcCore.java

```diff
@@ -331,7 +331,7 @@ public Path getExerciseChecksumCacheLocation() {
     public ListenableFuture<URI> pasteWithComment(Path path, String comment)
             throws TmcCoreException {
         //checkParameters(path);
```

Comment:
This should have been removed earlier (hups :D ), get rid of it. No commented code.